aptos-move/framework/supra-framework/sources/automation_registry.move

```diff
@@ -326,36 +218,28 @@ module supra_framework::automation_registry {
 
     #[view]
     /// Returns next task index in registry
-    public fun get_next_task_index(): u64 acquires AutomationRegistry {
-        let automation_registry = borrow_global<AutomationRegistry>(@supra_framework);
-        automation_registry.current_index + 1
+    public fun get_next_task_index(): u64 {
+        automation_registry_state::get_next_task_index()
     }
 
     #[view]
     /// List all the automation task ids
-    public fun get_active_task_ids(): vector<u64> acquires AutomationRegistry {
-        let automation_registry = borrow_global<AutomationRegistry>(@supra_framework);
-
-        let active_task_ids = vector[];
-        let ids = enumerable_map::get_map_list(&automation_registry.tasks);
-
-        vector::for_each(ids, |id| {
-            let task = enumerable_map::get_value(&automation_registry.tasks, id);
-            if (task.is_active) {
-                vector::push_back(&mut active_task_ids, id);
-            };
-        });
-        return active_task_ids
+    public fun get_active_task_ids(): vector<u64> {
+        automation_registry_state::get_active_task_ids()
     }
 
     #[view]
     /// Retrieves the details of a automation task entry by its ID.
     /// Returns a tuple where the first element indicates if the registry is completed/failed (`true`) or pending (`false`),
     /// and the second element contains the `AutomationTaskMetaData` details.
```

Comment:
Actually, I forgot to update the comment after making changes to the response type